### ISSUE TITLE
Getting skin.min.css net::ERR_ABORTED 404 (Not Found) in Production

### ISSUE THREAD

**Issue by basharabdullah** [TrestleAdmin/trestle-tinymce#6]
I have this weird issue where locally, using TinyMCE works fine, but in production, it doesn't load, and I see this error in web console

```
tinymce-2c7206a818d93b66cd975d66eb1bb2d59aa4d069f63f021e53820ebdf9a3129d.js:4866 GET https://example.com/assets/tinymce/skins/trestle/content.min.css net::ERR_ABORTED 404 (Not Found)
tinymce-2c7206a818d93b66cd975d66eb1bb2d59aa4d069f63f021e53820ebdf9a3129d.js:4866 GET https://example.com/assets/tinymce/skins/trestle/skin.min.css net::ERR_ABORTED 404 (Not Found)
```

The urls in production do not load indeed, but they do locally. 

Anyone face similar problem?

**Comment by rafamanzo** [comment]
Similar error here. In my case it works under development because the assets are not precompiled, but fails under production because these assets are not listed for precompilation.

Workaround, working in my Heroku app, adding to `config/initializers/assets.rb` the following:

```ruby
Rails.application.config.assets.precompile += %w( tinymce/skins/trestle/content.min.css tinymce/skins/trestle/skin.min.css tinymce/skins/trestle/fonts/tinymce.woff tinymce/skins/trestle/fonts/tinymce.ttf )
```

The maintainer may correct me if I'm wrong, but I think the definitive fix is modifying https://github.com/TrestleAdmin/trestle-tinymce/blob/master/lib/trestle/tinymce/engine.rb#L6 to:

```ruby
  config.assets.precompile << %w( trestle/tinymce.js tinymce/skins/trestle/content.min.css tinymce/skins/trestle/skin.min.css  tinymce/skins/trestle/fonts/tinymce.woff tinymce/skins/trestle/fonts/tinymce.ttf )
```

**Comment by spohlenz** [comment]
Thank you both for reporting this issue. This turned out to be a compatibility issue with Sprockets 4 which required an extra manifest file.

I've released gems for trestle-tinymce 0.2.1 which tracks TinyMCE 4.x and trestle-tinymce 0.3.0 for TinyMCE 5.x.
